web/src/components/njwds-layout/SidebarPageLayout.tsx

```diff
@@ -23,45 +24,52 @@ export const SidebarPageLayout = ({
   outlineBox = true,
   stackNav,
   belowBoxComponent,
+  titleOverColumns,
 }: SidebarPageLayoutProps): ReactElement => {
   const isLargeScreen = useMediaQuery(MediaQueries.desktopAndUp);
   const { Config } = useConfig();
 
   const backButton = (
```

Comment:
[pebble] This should be its own component.

Reply:
You mean like a component passed instead of a string?

Or do you mean that this should be its own component outside of this component?

Reply:
got it, will make back button its own component